public/js/app/pages/CreateNewMemo.jsx

```diff
@@ -108,12 +108,11 @@ function CreateNewMemo(props) {
     })
   }
 
-  const onChoiceOfSemester = event => {
-    const newSemester = event.target.value
+  const onChoiceOfSemester = useCallback(selectedSemester => {
     const { existingDraftEndPoint } = chosen
-    setSemester(newSemester)
+    setSemester(selectedSemester)
     _cleanUpPrevCheckboxesState(existingDraftEndPoint)
-  }
+  }, [])
```

Comment:
Is eslint complaining about `_cleanUpPrevCheckboxesState` missing from the dep-array?